storage/txcache/score.go

```diff
@@ -93,17 +84,9 @@ func computeSenderScore(params senderScoreParams) float64 {
 	return score
 }
 
-// GetScoreChunk returns the score chunk the sender is currently in
-func (listForSender *txListForSender) GetScoreChunk() *maps.MapChunk {
-	listForSender.scoreChunkMutex.RLock()
-	defer listForSender.scoreChunkMutex.RUnlock()
-
-	return listForSender.scoreChunk
+type disabledScoreComputer struct {
```

Comment:
When do we use this disabled score computer?

Reply:
Missed that - fixed (moved to test utilities).